Controllers/OI/Joysticks.java

```diff
@@ -20,7 +20,7 @@ public static Joysticks getInstance() {
 
     public static final class OIConstants  {
         public static final int DRIVE_PORT = 0;
-        public static final int TRANS_PORT = 0;
+        public static final int TRANS_PORT = 1;
```

Comment:
considering that we are moving to one xbox controller for all robot movement & one for operator controls, do we need a separate translation port?

Reply:
thats for the joysticks class, assuming we ever want to use joysticks it allows us to switch easily using the OI base class